src/Tribe/Views/V2/Template/JSON_LD.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Handles JSON-LD for V2.
+ *
+ * @since TBD
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+
+namespace Tribe\Events\Views\V2\Template;
+
+/**
+ * Class JSON_LD
+ *
+ * @since TBD
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+class JSON_LD {
+
+	/**
+	 * Fires to add JSON LD to Single Event
+	 *
+	 * @since TBD
+	 *
+	 */
+	public function print_single_json_ld() {
+
+		// Check if we are in a single page.
+		if ( ! is_singular( \Tribe__Events__Main::POSTTYPE ) ) {
+			return;
+		}
+
+		$context = tribe_context();
+
+		// One more check for our Post Type.
+		if ( ! $context->is( 'tec_post_type' ) ) {
+			return;
+		}
+
+		// Bail when that action already exists.
+		if ( has_action( 'wp_head', [ \Tribe__Events__JSON_LD__Event::instance(), 'markup' ] ) ) {
+			return;
+		}
+		log_me('here');
+		// Print JSON-LD markup on the`wp_head`.
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening bracket; 0 found
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: a9fad6ed42027138b6ad8efd02e2a5f1</sub>